lib/APPRemoteControl/src/SerialMuxChannels.h

```diff
@@ -205,6 +211,13 @@ typedef struct _LineSensorData
     uint16_t lineSensorData[5U]; /**< Line sensor data [digits] normalized to max 1000 digits. */
 } __attribute__((packed)) LineSensorData;
 
+/** Struct of the "Turtle" channel payload. */
+typedef struct _TurtleSpeed
+{
+    int32_t linearCenter; /**< Linear speed of the vehicle center. [mm/s] */
+    int32_t angular;      /**< Angular speed. [mrad/s] */
+} __attribute__((packed)) TurtleSpeed;
```

Comment:
Couldn't that be combined with  SpeedData?
SpeedData comes in with mm/s according to structure definition, but its not converted to steps/s!
And SpeedData parameter "center" is not used.

Reply:
It could be combined. I decided against it in order to maintain SpeedData untouched for the other applications that use it. Should I then add the angular speed to SpeedData?

Reply:
Hmm ... shall we provide two channels?
1. linear speed left and right (no center, no angular)
2. linear speed center + angular speed

And both in mm/s?

Reply:
The second channel is this "TurtleSpeed" channel. At least that was the idea

Reply:
I would keep RemoteControl as generic as possible. This way other applications can use the interfaces without the necessity to define their own channel. That means we have two channels, one is dealing with motor speeds (SpeedData) and the other with robot speeds (TurtleSpeed).

It makes sense to rename the channels and the callbacks, to show this to the user.
E.g. MotorSpeed / RobotSpeed channels.